data/json/recipes/recipe_deconstruction.json

```diff
@@ -701,38 +701,6 @@
     "qualities": [ { "id": "SCREW", "level": 1 } ],
     "components": [ [ [ "burnt_out_bionic", 1 ] ] ]
   },
-  {
-    "result": "bikini_bottom",
-    "type": "uncraft",
-    "time": "1 m",
-    "activity_level": "NO_EXERCISE",
-    "qualities": [ { "id": "CUT_FINE", "level": 1 } ],
-    "components": [ [ [ "sheet_lycra", 10 ] ], [ [ "thread_nomex", 1 ] ], [ [ "thread", 3 ] ] ]
-  },
-  {
-    "result": "bikini_top",
-    "type": "uncraft",
-    "time": "2 m",
-    "activity_level": "NO_EXERCISE",
-    "qualities": [ { "id": "CUT_FINE", "level": 1 } ],
-    "components": [ [ [ "sheet_lycra", 10 ] ], [ [ "thread_nomex", 1 ] ], [ [ "thread", 6 ] ] ]
-  },
-  {
-    "result": "bikini_top_fur",
-    "type": "uncraft",
-    "time": "20 s",
-    "activity_level": "NO_EXERCISE",
-    "qualities": [ { "id": "CUT_FINE", "level": 1 } ],
-    "components": [ [ [ "fur", 1 ] ] ]
-  },
-  {
-    "result": "bikini_top_leather",
-    "type": "uncraft",
-    "time": "20 s",
-    "activity_level": "NO_EXERCISE",
-    "qualities": [ { "id": "CUT_FINE", "level": 1 } ],
-    "components": [ [ [ "leather", 1 ] ] ]
-  },
```

Comment:
Pretty sure you don't need to delete these uncraft recipes for this PR

Reply:
Oh damn, thanks for the find. Fixing now